tensorflow/core/kernels/xent_op.cc

```diff
@@ -87,15 +95,16 @@ class SoftmaxXentWithLogitsOp : public OpKernel {
     // Try to reuse the logits_in buffer for the backprop output.
     OP_REQUIRES_OK(context, context->forward_input_or_allocate_output(
                                 {0}, 1, shape_in, &back_out));
-    if (shape_in.dim_size(0) > 0) {
-      functor::XentFunctor<Device, T> functor;
-      functor(context->eigen_device<Device>(), shape_in.AsEigenDSizes<2>(),
-              BCast::ToIndexArray<2>(bcast.x_bcast()),
-              BCast::ToIndexArray<2>(bcast.y_bcast()),
-              logits_in.template shaped<T, 2>(bcast.x_reshape()),
-              labels_in.template shaped<T, 2>(bcast.y_reshape()),
-              scratch.matrix<T>(), loss_out->vec<T>(), back_out->matrix<T>());
-    }
+
```

Comment:
What changed in this block?

Reply:
It seems to have no impact. I reversed it to previous version.